.lintr

```diff
@@ -5,6 +5,21 @@ exclusions: list(
     "R/meas-lcdm-crum.R" = list(
       object_usage_linter = Inf
     ),
+    "R/meas-ncrum.R" = list(
+      object_usage_linter = Inf,
+      indentation_linter = Inf
+    ),
+    "R/meas-nida.R" = list(
+      object_usage_linter = Inf,
+      indentation_linter = Inf
+    ),
+    "R/meas-nido.R" = list(
+      object_usage_linter = Inf,
+      indentation_linter = Inf
+    ),
+    "R/utils-get-parameters.R" = list(
+      indentation_linter = Inf
+    ),
```

Comment:
I don't think we should be ignoring these lints--is there a reason we can't fix the code to comply?

Reply:
I'm working on the indentation_linter exception. I've had problems with that one in the past, but I'll to get them resolved here. The object_usage_linter is included because lintr is not recognizing that defined objects are being used within a `glue()` statement. We have a similar lint exception for meas-lcdm-crum.R, where we tell it to not look for object usage.

Reply:
for my own tracking, this has been corrected